src/services/translation.js

```diff
@@ -1,20 +1,16 @@
 import i18n from 'i18next';
+import i18NextHttpBackend from 'i18next-http-backend';
+import {initReactI18next} from 'react-i18next';
 
-export const init = ({lang}) => {
-    return new Promise(async (resolve, reject) => {
-        const response = await fetch(`/locales/${lang}/strings.json`);
-        const translation = await response.json();
-
-        i18n.init({
-            lng: lang,
-            resources: {
-                [lang]: {translation}
-            }
-        }, (err, t) => {
-            if (err) return reject(err);
-
-            global.t = t;
```

Comment:
No longer need to store `t` globally - due to the fact that `react-i18next` exposes both the translation function and the entire i18n object.